### PR TITLE
[WebApp SALAD-23203] Replace average earning columns with only average earning rate for demand page table

### PR DESCRIPTION
**Pull request by rners01**
image

### CODE REVIEW

File: packages/web-app/src/modules/demand-monitor-views/DemandMonitorPage/DemandMonitorTable/DemandMonitorTable.tsx [hunk 1]
```diff
@@ -193,6 +193,7 @@ const _DemandMonitorTable: FunctionComponent<Props> = ({ classes, demandedHardwa
               const demand = getHardwareDemandLevel(utilizationPct)
               const avgEarningTimeHours = earningRates.avgEarningTimeMinutes / 60
               const avgRunningTime = Math.round(avgEarningTimeHours * 10) / 10
+              const avgEarningRate = avgRunningTime && earningRates.avgEarning / avgRunningTime
```
Comment (vitto-moz) [inline packages/web-app/src/modules/demand-monitor-views/DemandMonitorPage/DemandMonitorTable/DemandMonitorTable.tsx:196]: we will have this data calculated on the BE, so no need for FE calculations

Comment (rners01) [reply packages/web-app/src/modules/demand-monitor-views/DemandMonitorPage/DemandMonitorTable/DemandMonitorTable.tsx]: removed and utilized `avgEarning`

File: packages/web-app/src/modules/demand-monitor-views/DemandMonitorPage/DemandMonitorTable/DemandMonitorTable.tsx
Comment (Maks19) [inline packages/web-app/src/modules/demand-monitor-views/DemandMonitorPage/DemandMonitorTable/DemandMonitorTable.tsx:1]: Shouldn't `earningRates.avgEarning` be updated with a new name and values from the backend?

Comment (rners01) [reply packages/web-app/src/modules/demand-monitor-views/DemandMonitorPage/DemandMonitorTable/DemandMonitorTable.tsx:1]: about to check with @divanitskiy

**Review by Maks19 — APPROVED** [review]

**Review by Maks19 — APPROVED** [review]

**Pull request merged**